File: ViewModels/CaptureLockRequestEventArgs.vb
Namespace ViewModels
    Public Class CaptureLockRequestEventArgs
        Inherits EventArgs

        Private ReadOnly _reason As CaptureUnlockReason

        Public Sub New(reason As CaptureUnlockReason)
            _reason = reason
        End Sub

        Public ReadOnly Property Reason As CaptureUnlockReason
            Get
                Return _reason
            End Get
        End Property
    End Class
End Namespace

File: src/ClickSyncMouseTester/ViewModels/CaptureLockRequestEventArgs.cs
using System;

namespace ClickSyncMouseTester.ViewModels;

public class CaptureLockRequestEventArgs : EventArgs
{
    private readonly CaptureUnlockReason _reason;

    public CaptureUnlockReason Reason => _reason;

    public CaptureLockRequestEventArgs(CaptureUnlockReason reason)
    {
        _reason = reason;
    }
}





